src/rules/no-identical-functions.ts

```diff
@@ -151,4 +151,25 @@ const rule: TSESLint.RuleModule<string, Options> = {
   },
 };
 
+function isApplicable(functionNode: FunctionNode) {
```

Comment:
I think we can replace this function by using appropriate listeners in the rule `create` method

Reply:
I have replaced the filtering function by selectors as suggested.